### DISCUSSION TITLE
0.0.4

### DISCUSSION THREAD

**Discussion by umnik1** [umnik1/yaamp#70] [Announcements]
**Изменения:**

- Добавлена поддержка локальных .mp3 файлов
- Добавлено сохранение эквалайзера

Для того чтобы сохранить эквалайзер, нажмите кнопку "Presets" и "Save". Теперь при следующих заходах эквалайзер будет выставлять параметры такими как вы их настроили.

![image](https://github.com/umnik1/yaamp/assets/2902730/a0142ebc-517e-4f4c-b2aa-039ce7df96f9)

<hr /><em>This discussion was created from the release <a href='https://github.com/umnik1/yaamp/releases/tag/v0.0.4'>0.0.4</a>.</em>

**Comment by ikocs** [comment]
А обновить прям из приложения не выйдет? Только качать и ставить поверх новую версию?

**Reply by umnik1** [reply]
Извиняюсь за долгий ответ, да, нужно будет заново установить...

**Reply by Bubblegumovna** [reply]
Привет! у меня окно сломалось, я не знаю, как это исправить( Пробовала несколько раз переустанавливать, не помогло(
![photo_2025-01-11_14-58-13](https://github.com/user-attachments/assets/8b2e6d7f-ecc6-4b99-a4f2-74d0102fe9a3)

**Reply by umnik1** [reply]
> Привет! у меня окно сломалось, я не знаю, как это исправить( Пробовала несколько раз переустанавливать, не помогло( ![photo_2025-01-11_14-58-13](https://private-user-images.githubusercontent.com/194581362/402232136-8b2e6d7f-ecc6-4b99-a4f2-74d0102fe9a3.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzY1OTY2MzUsIm5iZiI6MTczNjU5NjMzNSwicGF0aCI6Ii8xOTQ1ODEzNjIvNDAyMjMyMTM2LThiMmU2ZDdmLWVjYzYtNGI5OS1hNGYyLTc0ZDAxMDJmZTlhMy5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwMTExJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDExMVQxMTUyMTVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iODA3Mzk4ZWY1MjNiYzA5NTZmMWM3ZTRkNGY1ZGNkNGI5YzZjYjAxNDQ5ZGEzNTI2YTY3NGU5NjFlZDg5OGRmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.QrTrgEBm_AgaU-FXoT5pV9pWhbDC_C6DgkG87WobjKE)

Привет, попробуй удалить файлы **settings.json** и **skin.json** из папки "**C:\Users\_<имя пользователя>_\AppData\Roaming\yaamp**"

**Reply by Bubblegumovna** [reply]
Большое спасибо, помогло =)
